roles/valet-init/templates/valet-sh.yml.j2

```diff
@@ -0,0 +1,58 @@
+## experimental feature that is used by "init-instance" and "restore" command
+hub:
+# the hostname
+  host: "dummy.example.com"
+# the ssh target port
+  port: 22
+# the absolute path
+  path: "/data"
+
+services:
+  node:
+    version: 10
+  php:
+    # php version that should be used in the nginx configuration but for cli execution as well
+    version: 7.3
```

Comment:
I would set the php version to 7.4 per default as the init project will most likely be used most for new projects